src/components/MainContainer.js

```diff
@@ -294,6 +291,16 @@ class MainContainer extends Component {
         window.removeEventListener('resize', this.updateMapContainerDimensions)
     }
 
+    instantiateScenarios(dataset) {
```

Comment:
This has no references to `this`. This can be static and extracted out of the file. Also makes it easier to unit test